lib/WorkflowMain.groovy

```diff
@@ -9,8 +9,6 @@ class WorkflowMain {
     //
     public static String citation(workflow) {
         return "If you use ${workflow.manifest.name} for your analysis please cite:\n\n" +
-            "* The pipeline publication\n" +
-            "  https://doi.org/10.1021/acs.jproteome.9b00313\n\n" +
```

Comment:
Why do you remove the citation? Is is not wanted here?

Reply:
Good find, that should still be included indeed